src/rp2_common/hardware_irq/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ pico_simple_hardware_target(irq)
 
 target_sources(hardware_irq INTERFACE ${CMAKE_CURRENT_LIST_DIR}/irq_handler_chain.S)
 
-target_link_libraries(hardware_irq INTERFACE pico_sync) # not mirrored as only implementation requires it
+target_link_libraries(hardware_irq INTERFACE pico_sync pico_runtime_init_headers pico_runtime) # not mirrored as only implementation requires it
```

Comment:
why is `pico_runtime` here?

Reply:
Presumably because of this?
```
/home/peterh/source/pico2/pico-sdk-rp2350/src/rp2_common/pico_runtime_init/include/pico/runtime_init.h:11:10: fatal error: pico/runtime.h: No such file or directory

   11 | #include "pico/runtime.h"

      |          ^~~~~~~~~~~~~~~~


```

Reply:
yeah; i was concerned about it pulling in `pico_runtime`, but yeah `pico_runtime_init_headers` should pull in `pico_runtime_headers`